call-a-function-api-oci-clouddev.md

```diff
@@ -0,0 +1,772 @@
+
+### Create Functions Application
+
+To create application, follow these steps.
+
+1. Open the navigation menu and click **Developer Services**. Under **Functions**, click **Applications**.
+2. Select your compartment from the **Compartment** drop-down.
+3. Click **Create Application**.
+4. Fill in the form data.
+    * **Name:** `<your-app-name>`
+    * **VCN:** `<your-vcn-name>`
+    * **Subnets:** `<Public-Subnet-your-vcn-name>`
+5. Click Create.
+
+Your app is created.
+
+### Setup Ingress Rule for HTTPS
+
+1. Open the navigation menu and click **Networking**, and then click **Virtual Cloud Networks**.
+2. Click the name of the VCN you used to for your Oracle Functions application.
+3. With your new VCN displayed, click your **Public** subnet link. 
+
+    The public subnet information is displayed with the Security Lists at the bottom of the page.
+
+4. Click the Default Security List link or appropriate security list link. 
+
+    The default **Ingress Rules** for your VCN are displayed.
+
+5. Click Add Ingress Rules. 
+
+    An **Add Ingress Rules** dialog is displayed.
+
+6. Fill in the ingress rule with the following information. After all the data is entered, click Add Ingress Rules
+
+    Fill in the ingress rule as follows:
+
+    * **Stateless:** Checked
+    * **Source Type:** CIDR
+    * **Source CIDR:** 0.0.0.0/0
+    * **IP Protocol:** TCP
+    * **Source port range:** (leave-blank)
+    * **Destination Port Range:** 443
+    * **Description:** VCN for applications
+
+    After you click Add Ingress Rule, HTTPS connections are allowed to your **public subnet**.
+
+### Setup Policy for API Gateway Access to Functions
+
+Next, set up a policy which allows API Gateway to invoke functions.
+
+First, create a Dynamic Group for API Gateway.
+
+1. Open the navigation menu and click **Identity & Security**. Under **Identity**, click **Dynamic Groups**.
+2. Click **Create Dynamic Group**.
+3. Fill in the following information to define your dynamic group. 
+    * **Name:** `<name-for-your-dynamic-group>`
+    * Under **Matching Rules** use **Rule 1:** `<the-rule-text>`
+
+    Here is sample name and the rule you need to fill out.
+
+    * **Name:** api-gtw-func-dynamic-group
+    * Under **Matching Rules** use **Rule 1:** `ALL {resource.type = 'ApiGateway', resource.compartment.id = 'ocid1.compartment.<your-compartment-OCID>'}`
+4. Click **Create**.
+
+Now create the policy for API Gateway.
+
+1. Open the navigation menu and click **Identity & Security**. Under **Identity**, click **Policies**.
+2. Click **Create Policy**.
+3. To define your policy, fill in the following information.
+    * **Name:** `<name-for-your-policy>`
+    * **Description:** `<description-for policy>`
+    * **Compartment:** `<name-of-functions-compartment>`
+
+    For the Policy Builder Section
+
+    * Click the **Customize(Advanced)** link.
+    * Enter your policy in the text box, for example: 
+
+    ```console
```

Comment:
This is a child of the `for example:` line and should be indented another 4 spaces (along with `{:.notice}`